albumentations/augmentations/functional.py

```diff
@@ -910,8 +910,8 @@ def channel_dropout(img, channels_to_drop, fill_value=0):
 def gamma_transform(img, gamma):
     if img.dtype == np.uint8:
         invGamma = 1.0 / gamma
-        table = np.array([((i / 255.0) ** invGamma) * 255 for i in np.arange(0, 256)]).astype("uint8")
-        img = cv2.LUT(img, table)
+        table = ((np.arange(0, 256) / 255) ** invGamma) * 255
```

Comment:
It looks like `((np.arange(0, 256 / 255 , 1.0 / 255) ** invGamma) * 255)` is slightly faster.

Reply:
done